lab-elizabeth/README.md

```diff
@@ -0,0 +1,59 @@
+##About
+
+This is an HTTP server set up to give back messages based on requests by the user!
+
+you can
+
+##Setting up a server
+
+you need to have Node installed, of course.
+
+You can go to https://nodejs.org/ download from there, extract the file and install.
+
+Then, you need to go to the directory containing the server.js file and open it using node with the command `PORT=<your-port-number> node server.js` this will automatically start your server in the port you choose, you can also choose to enter just `node server.js` and it will automatically open the server on port 3000 for you.
+
+to exit the server, you simply need to hold ctrl and press C.
+
+##Making requests to the Server
+
+You will need to make sure that you have HTTPie installed:
+
+You can go to https://httpie.org/ and follow their directions.
+
+You can make GET requests to see text from the server by typing in:
+`http GET :<server port number>/cowsay text=="<whatever you want it to say here>"`
+
+You can make POST requests by typing-in:
+`http POST :<server port number>/cowsay text=="<whatever you want it to say here>"`
+
+You will exit the server automatically each time you make a request.
```

Comment:
It's not necessary to say this. By the nature of HTTP, connections are terminated after the request/response cycle. :D